edweiss-app/app/(app)/deck/[id]/index.tsx

```diff
@@ -200,7 +228,8 @@ const CardListScreen: ApplicationRoute = () => {
 					textColor='crust'
 					onPress={() => {
 						setShowDropdown(false);
-						router.push({ pathname: `/deck/${id}/card/creation` as any, params: { deckId: id } })
+						//router.push({ pathname: `/deck/${id}/card/creation` as any, params: { deckId: id } })
+						router.push({ pathname: `/deck/${id}/card/` as any, params: { deckId: id, mode: "Create", prev_question: "", prev_answer: "", cardIndex: "None" } })
```

Comment:
Here you should use `pushWithParameters`form `edweiss-app/hooks/routeParameters.ts` 
To retrieve the parameters, use `useRouteParameters`. It allows automatic parsing and sanitization

Reply:
Ohhh that sounds great. I don't know about this functionality yet. Thanks for telling me. It will definitely make the code looks better and more straightforward.